examples/platform/silabs/SiWx917/SiWx917/wfx_rsi.h

```diff
@@ -77,9 +84,9 @@ struct wfx_rsi
     sl_wfx_mac_address_t ap_bssid; /* To which our STA is connected */
     uint16_t join_retries;
     uint8_t ip4_addr[4]; /* Not sure if this is enough */
-};
+} wfx_rsi_t;
```

Comment:
Doesn't match with the previous CamelCase declaration... shouldn't it be WfxRsi_t ?